### PR TITLE
Office queue "updated at" field now shows non-zero date

### PR DESCRIPTION
**Pull request by breanneboland**
## Description

This changes the query line that fills the updated_at field to pull from the moves table instead of plopping "current_date" in there. 

## Reviewer Notes

Look ok? 

## Code Review Verification Steps

* [ ] All tests pass.
* [ ] Code follows the guidelines for [Logging](https://github.com/transcom/mymove/blob/master/docs/backend.md#logging)
* [ ] The requirements listed in
 [Querying the Database Safely](https://github.com/transcom/mymove/blob/master/docs/backend.md#querying-the-database-safely)
 have been satisfied.
* [ ] Any migrations/schema changes also update the diagram in docs/schema/dp3.sqs
* [ ] There are no aXe warnings for UI.
* [ ] This works in IE.
* Any new client dependencies (Google Analytics, hosted libraries, CDNs, etc) have been:
  * [ ] Communicated to @willowbl00
  * [ ] Added to the list of [network dependencies](https://github.com/transcom/mymove#client-network-dependencies)
* [ ] Request review from a member of a different team.
* [ ] (TRIAL) Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/story/show/157617014) for this change
* [this article](https://en.wikipedia.org/wiki/0) explains more about the approach used.

## Screenshots
screen shot 2018-06-11 at 4 59 49 pm

### CODE REVIEW

**Comment by jim** [comment]
@breanneboland Should we also replace the `current_date` in that query?

**Comment by breanneboland** [comment]
@jim, I like that idea. I added that in this last commit.

**Comment by kilbergr** [comment]
Some discovery:
Failing on `TestShowQueueHandler` for both statuses. I think it's because you don't have a PPM set up happening in the test, so there's no PPM table to join in the query.

File: pkg/models/queue.go [hunk 1]
```diff
@@ -38,13 +38,14 @@ func GetMoveQueueItems(db *pop.Connection, lifecycleState string) ([]MoveQueueIt
 				CONCAT(COALESCE(sm.last_name, '*missing*'), ', ', COALESCE(sm.first_name, '*missing*')) AS customer_name,
 				moves.locator as locator,
 				ord.orders_type as orders_type,
-				current_date as move_date,
+				ppm.planned_move_date as move_date,
```
[the remark this answers is not in the document]
Comment (jim) [reply pkg/models/queue.go]: This is correct. I will update the query to use `LEFT JOIN` to account for this.

**Pull request merged**